pyproject.toml

```diff
@@ -23,6 +23,7 @@ posthog = "^3.5.0"
 groq = "^0.9.0"
 together = "^1.2.1"
 boto3 = "^1.34.144"
+litellm = "^1.41.24"
```

Comment:
Lets make all of these as non-mandatory dependency on mem0ai similar to how we did for Embedchain.

Reply:
Actually feel free to do that as part of a follow up PR. Happy to merge this one.

Reply:
Yes will make those changes in next PR